lib/OnboardingMessageProvider.jsm

```diff
@@ -53,11 +53,11 @@ const ONBOARDING_MESSAGES = async () => ([
     bundled: 3,
     order: 2,
     content: {
-      title: {string_id: "onboarding-private-browsing-title"},
```

Comment:
Let's leave those as `string_id` to match other cards

Reply:
Reverted these changes